### DISCUSSION TITLE
How can I save template with tags like gjs-html,gjs-components,gjs-assets,gjs-css,gjs-styles?

### DISCUSSION THREAD

**Discussion by wyxcoder** [GrapesJS/grapesjs#5130] [Q&A]
My existing code save template with tags like gjs-html,gjs-components,gjs-assets,gjs-css,gjs-styles. But with current version, the saved json has tags like assets,styles,pages,fonts. Is that the new way to save? How can I save with the tags starts with gjs-?

**Comment by artf** [comment]
Yes, this is an update like from a year ago... read the [Storage](https://grapesjs.com/docs/modules/Storage.html) documentation please.

**Reply by RakulAgn** [reply]
i am also facing same issue right now So its not possible to do it in older method  we have to migrate it right ?

**Reply by RakulAgn** [reply]
i have lot of data in older method is there any way to migrate it to new method
